src/pages/home/HeaderView.js

```diff
@@ -74,6 +76,43 @@ const HeaderView = (props) => {
     // We hide the button when we are chatting with an automated Expensify account since it's not possible to contact
     // these users via alternative means. It is possible to request a call with Concierge so we leave the option for them.
     const shouldShowCallButton = (isConcierge && guideCalendarLink) || !isAutomatedExpensifyAccount;
+    const shouldShowThreeDotsButton = isTaskReport;
+    const threeDotMenuItems = [];
+
+    if (shouldShowThreeDotsButton) {
+        if (props.report.stateNum === CONST.REPORT.STATE_NUM.OPEN && props.report.statusNum === CONST.REPORT.STATUS.OPEN) {
+            threeDotMenuItems.push({
+                icon: Expensicons.Checkmark,
+                text: props.translate('newTaskPage.markAsComplete'),
+
+                // Implementing in https://github.com/Expensify/App/issues/16858
+                onSelected: () => {},
+            });
+        }
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/19631

This introduces a regression where the user non-assignee, or not owner, can still see the action button and press it.